README.md

```diff
@@ -1,47 +1,44 @@
-# Minetest Pick_Axe_Tweaks Mod
-
+# Minetest Pickaxe Tweaks Mod
 ![Luacheck](https://github.com/wsor4035/pick_axe_tweaks/workflows/luacheck/badge.svg)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+[![ContentDB](https://content.minetest.net/wsor4035/pick_axe_tweaks/shields/downloads/)](https://content.minetest.net/packages/wsor4035/pick_axe_tweaks/)
 
 ## About
 
-adds the ability for pick axes to place a configurable light node ffrom in game and per player
+Adds the ability for pickaxes to place light nodes. Configurable per player.
 
 ## Links
 
 * [Github](https://github.com/wsor4035/pick_axe_tweaks)
-* [Contentdb](not yet available)
-* [forums](not avaible yet)
+* [ContentDB](https://content.minetest.net/packages/wsor4035/pick_axe_tweaks/)
+* Minetest forum (not avaible yet)
 
 ## Setup
 
-download mod, put in your worldmods folder, or mods folder and enable it
-* if not using with mtg, please change the setting found under header `minetest.conf settings`
-* also use a mod that uses this mods api to register its pick axes
+Download mod, put in your `mods` or `worldmods` folder and enable it.
+
+If not using with Minetest Game, please change the default light node setting.
 
-## minetest.conf settings 
+## Settings 
 
-* pat_light_node (by default default:torch): default light node for pick axes
+* `pat_light_node`: Default light node. (by default `default:torch`)
 
 ## API
 
-example pulled from init.lua
+`pick_axe_tweaks.register_pick_axes()`: Registers a list of pickaxes.
+
+Example:
 
 ```lua
-if minetest.get_modpath("default") then
-    local default_pick_axes = {
-        "default:pick_wood",
-        "default:pick_stone",
-        "default:pick_bronze",
-        "default:pick_steel",
-        "default:pick_mese",
-        "default:pick_diamond",
-    }
-
-    pick_axe_tweaks.register_pick_axes(default_pick_axes)
-end
+local pickaxes = {
+	"my_mod:pick_wood",
+	"my_mod:pick_stone",
+	"my_mod:pick_diamond",
+}
+
+pick_axe_tweaks.register_pick_axes(pickaxes)
```

Comment:
probably should encourage just adding the pickaxe group to item, and as a side note mention that if not, use this single code line of something.....

Reply:
Done. I replaced `pick_axe_tweaks.register_pick_axes(pickaxes)` with `pick_axe_tweaks.register_tool(name)`, because now the API is only used for non-pickaxes.

Reply:
should add `pick_axe_tweaks.register_pick_axes = pick_axe_tweaks.register_tool` for backwards compat

Reply:
https://github.com/mt-mods/pick_axe_tweaks/blob/dabe5b115789512eeaba872e03c73a27b7a519a0/api.lua#L67-L75